package.json

```diff
@@ -43,7 +43,7 @@
     "typescript": "^4.1.5"
   },
   "scripts": {
-    "start": "PORT=3001 react-scripts start",
```

Comment:
We don’t need this port, right?

Reply:
Default port is `3000`. And if you would like to use another port this is the way to do it. Be aware that Keycloak also needs to whitelist the host including port e.g. `http://localhost:3000` to allow for logging in when working locally. I believe only `http://localhost:3000` is currently whitelisted. So I would not use this. And use the default port for both `npm start` and `npm start:local`.